docs/src/componentDocs/Chip/playground/PlaygroundPage.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Chip, ChipProps } from '@brightlayer-ui/react-native-components';
+import {
+    InputConfig,
+    PreviewComponent,
+    CodeSnippetFunction,
+    getPropsToString,
+    getPropsMapping,
+    Playground,
+} from '@brightlayer-ui/react-doc-components';
+import { IconSource } from '@brightlayer-ui/react-native-components/core/__types__';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import { Avatar } from 'react-native-paper';
+
+const inputConfig: InputConfig = [
+    // Optional Props
+    {
+        id: 'chipColor',
+        type: 'color',
+        typeLabel: `string`,
+        initialValue: '',
+        description: 'The color of the chip',
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'textColor',
+        type: 'color',
+        typeLabel: `string`,
+        initialValue: '',
+        description: 'The color of the text label',
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'iconColor',
+        type: 'color',
+        typeLabel: `string`,
+        initialValue: '',
+        description: 'The color of the icon',
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'borderColor',
+        type: 'color',
+        typeLabel: `string`,
+        initialValue: '',
+        description: 'TThe color of border color',
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'icon',
+        type: 'select',
+        typeLabel: 'IconSource',
+        initialValue: 'info',
+        description: 'A component to render for the icon',
+        options: [
+            { label: 'undefined', value: 'undefined' },
+            { label: 'info', value: 'info' },
+        ],
+        required: false,
+        category: 'Optional Props',
+    },
+    {
+        id: 'mode',
+        type: 'string',
+        typeLabel: `string`,
+        initialValue: 'elevated',
+        description: 'The size of the circle in px',
```

Comment:
This description is not correct. We need to update API docs as well. 
Since modes are 2 `outlined` and `elevated` we can have type `select`.

Reply:
yes the type needs to be select
and description also doesn't look correct

Reply:
Updated the type as well as description.